npm-shrinkwrap.clean.json

```diff
@@ -207,7 +207,7 @@
           "dependencies": {
             "traceur": {
               "version": "0.0.33",
-              "resolved": "git+https://github.com/vojtajina/traceur-compiler#d90b1e34c799bf61cd1aafdc33db0a554fa9e617",
+              "resolved": "git://github.com/vojtajina/traceur-compiler.git#d90b1e34c799bf61cd1aafdc33db0a554fa9e617",
```

Comment:
To create this diff, I did `npm shrinkwrap --dev` and then `./scripts/npm/clean-shrinkwrap`. I'm not sure why this change happened. Does anyone know if I should just revert this line?